app/src/main/kotlin/com/mapzen/erasermap/controller/MainActivity.kt

```diff
@@ -652,6 +622,9 @@ class MainActivity : AppCompatActivity(), MainViewController,
      * to show or hide debug settings in settings fragment
      */
     override fun toggleShowDebugSettings() {
+        if (!AndroidAppSettings.SHOW_DEBUG_SETTINGS_QUERY.equals(searchController.searchView?.query.toString())) {
```

Comment:
Erg, that is still the case...what happened is that I refactored the kotlin syntax and then when resolving the rebase conflicts not realized that it had been moved. 

Removed.